### ISSUE TITLE
Could not find com.mikepenz:library-core:3.0.4

### ISSUE THREAD

**Issue by lixianyu** [mikepenz/Android-Iconics#407]
## About this issue

- Could not find com.mikepenz:library-core:3.0.4.
- Everytime when I build.

### All dependencies in gradle app:
```
dependencies {
    implementation fileTree(include: ['*.jar'], dir: 'libs')
    implementation 'com.android.support:appcompat-v7:27.1.1'
    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
    implementation 'com.android.support:support-v4:27.1.1'
    implementation 'com.android.support:design:27.1.1'
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
    //  For circle image view
    implementation 'de.hdodenhof:circleimageview:2.2.0'

//-----------------------------------------------------------------------------------------------
    //the core iconcis library (without any widgets)
    implementation "com.mikepenz:iconics-core:3.0.4"
//    implementation "androidx.appcompat:appcompat:${androidXVersion}"

    //implementation "com.mikepenz:iconics-views:3.1.0-rc01"
    implementation "com.mikepenz:iconics-views:3.0.4"

    // Choose your desired fonts
    implementation 'com.mikepenz:google-material-typeface:3.0.1.2.original@aar'
    implementation 'com.mikepenz:material-design-iconic-typeface:2.2.0.4@aar'
//    implementation 'com.mikepenz:fontawesome-typeface:5.0.6.0@aar'
//    implementation 'com.mikepenz:octicons-typeface:3.2.0.4@aar'
//    implementation 'com.mikepenz:meteocons-typeface:1.1.0.4@aar'
//    implementation 'com.mikepenz:community-material-typeface:2.0.46.1@aar'
//    implementation 'com.mikepenz:weather-icons-typeface:2.0.10.4@aar'
//    implementation 'com.mikepenz:typeicons-typeface:2.0.7.4@aar'
//    implementation 'com.mikepenz:entypo-typeface:1.0.0.4@aar'
//    implementation 'com.mikepenz:devicon-typeface:2.0.0.4@aar'
//    implementation 'com.mikepenz:foundation-icons-typeface:3.0.0.4@aar'
//    implementation 'com.mikepenz:ionicons-typeface:2.0.1.4@aar'
//    implementation 'com.mikepenz:pixeden-7-stroke-typeface:1.2.0.2@aar'
//-----------------------------------------------------------------------------------------------


    //PushSDK必须依赖基础组件库，所以需要加入对应依赖
    implementation 'com.umeng.sdk:common:1.5.3'
    //implementation 'com.umeng.sdk:common:latest.integration'

    //PushSDK必须依赖utdid库，所以需要加入对应依赖
    implementation 'com.umeng.sdk:utdid:1.1.5.3'
    //implementation 'com.umeng.sdk:utdid:latest.integration'

    //PushSDK
    implementation 'com.umeng.sdk:push:4.2.0'
    //implementation 'com.umeng.sdk:push:latest.integration'

    implementation 'com.umeng.sdk:analytics:7.5.3'
    //implementation 'com.umeng.sdk:analytics:latest.integration'
}
```

## The build error log
```
Could not find com.mikepenz:library-core:3.0.4.
Searched in the following locations:
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/google/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/google/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/android/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    file:/C:/Users/lxy/AppData/Local/Android/Sdk/extras/android/m2repository/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
    https://dl.google.com/dl/android/maven2/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    https://dl.google.com/dl/android/maven2/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
    https://jcenter.bintray.com/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    https://jcenter.bintray.com/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
    https://repo.maven.apache.org/maven2/com/mikepenz/library-core/3.0.4/library-core-3.0.4.pom
    https://repo.maven.apache.org/maven2/com/mikepenz/library-core/3.0.4/library-core-3.0.4.jar
Required by:
    project :app > com.mikepenz:iconics-views:3.0.4

```

**Comment by zTrap** [comment]
Try to use `3.0.4@aar` instead of `3.0.4`

**Comment by lixianyu** [comment]
@zTrap , @mikepenz , 
### Thank you very much! **It's work.**

**Comment by mikepenz** [comment]
Ok the pom file issue is fixed in v3.1.0-rc01 already :)
